src/build/cargo.rs

```diff
@@ -294,37 +293,25 @@ impl Executor for RlsExecutor {
             }
         }
 
-        let rustc_exe = env::var("RUSTC").unwrap_or(env::args().next().unwrap());
-        let mut cmd = Command::new(&rustc_exe);
+        // Cargo should handle `$RUSTC` wrapper override already
+        let mut cmd = cargo_cmd.clone();
```

Comment:
This will use the same program that's inside `cargo_cmd`, is it correct wrt shim (and `env::args().next()`)?